agent/src/beerocks/fronthaul_manager/ap_manager/ap_manager_thread.cpp

```diff
@@ -132,30 +132,26 @@ bool ap_manager_thread::init()
         return false;
     }
 
-    //connect to slave //
+    return true;
```

Comment:
If I remember correctly, it was because I planned to not call the init function from the fronthaul main file.
The only thing left there is  ap_hal command that calls to a shell command "iwpriv" which I'm not sure is still working.

Anyway, I didn't check it and left the init function for now.